pkg/db/connection_test.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var (
+	host     = "localhost"
+	port     = "5432"
+	user     = "postgres"
+	dbname   = "Hospital"
+	password = "root"
+	sslmode  = "disable"
+)
+
+func TestGetConnection(t *testing.T) {
+	assert := assert.New(t)
+	_, err := GetConnection(host, port, user, dbname, password, sslmode)
+	assert.NoError(err, fmt.Errorf("got an error when tried to make connection with database:%w", err))
+}
+
+func TestGetConnection2(t *testing.T) {
```

Comment:
bad name for the function. Let's try to renamed it to something better, or just add comments with description